integreat_cms/api/v3/feedback/imprint_page_feedback.py

```diff
@@ -68,13 +68,16 @@ def imprint_page_feedback_internal(
     :param is_technical: is feedback on content or on tech
     :type is_technical: bool
 
-    :raises ~django.http.Http404: HTTP status 404 if the region has no imprint
+    :raises ~django.http.Http404: HTTP status 404 if the region has no imprint or the selected language does not exist for this region
 
     :return: JSON object according to APIv3 imprint feedback endpoint definition
     :rtype: ~django.http.JsonResponse
     """
-    if region.imprint:
+    imprint = region.imprint
+
+    if imprint and language in region.languages:
         ImprintPageFeedback.objects.create(
+            imprint_page_translation=imprint.get_translation(language.slug),
```

Comment:
Just because `language` is in `region.languages` does not mean that this translation exists - and if it doesn't, this means that the API does not return a 404, but throws an internal server error instead (